### PR TITLE
readme enhancements to 2024 robotpy and pipenv

### PR DESCRIPTION
**Pull request by lmaxwell24**
This has got to be @LandonBayer favorite pr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,39 +114,43 @@ py -3 -m venv ./.venv
    ```bash
    pipenv shell
    ```
-   (examples: `robotpy`, `robotpy[ctre,navx]`, `robotpy[all]`) (see: [robotpy on pypi](https://pypi.org/project/robotpy/))
-1. **Download python for roboRIO**
+1. **download required robotpy packages**
+   This also includes the python image, as well as all packages defined in pyproject.toml
```
Comment (LandonBayer) [inline README.md:118]: Does this even have a pyproject.toml

File: README.md [hunk 2]
```diff
@@ -114,39 +114,43 @@ py -3 -m venv ./.venv
...
+
+   alternatively you can run the installer directly
+   (to download the python image)
+
    ```bash
-   python -m robotpy_installer download robotpy
+   pipenv run robotpy installer download-python
    ```
+
+   (to download the python dependancies)
```
Comment (LandonBayer) [inline README.md]: Dependencies

**Review by LandonBayer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lmaxwell24) [reply README.md:118]: with #9 it would
didn't feel like putting in an inaccurate pyproject.toml

File: README.md (hunk 2 above)
Comment (lmaxwell24) [reply README.md]: i can spell

**Review by LandonBayer — APPROVED** [review]

**Review by icai0 — APPROVED** [review]

**Pull request merged**